src/main/webapp/index.html

```diff
@@ -30,26 +30,31 @@
         <span id="place-name" class="title"></span><br />
         <span id="place-address"></span>
       </div>
-      <div id="infowindow-restaurant-content">
-        <div id="info-restaurant-name">${name}</div>
+      <div id="infowindow-restaurant-content" class="hide">
+        <div id="info-restaurant-name"></div>
         <a target="_blank" id="info-restaurant-dir">
-          <img src="https://gstatic.com/images/icons/material/system/2x/directions_gm_blue_20dp.png">
+          <span class="iconify" data-icon="bx:bxs-direction-right" data-inline="false" data-width="38px" data-height="38px" style="color: blue;"></span>
         </a>
         <a id="whatsapp-share">
-          <span class="iconify" data-icon="logos:whatsapp" data-inline="false"></span>
+          <span class="iconify" data-icon="logos:whatsapp" data-inline="false" data-width="40px" data-height="40px"></span>
         </a>
         <a id="telegram-share">
-          <span class="iconify" data-icon="logos:telegram" data-inline="false"></span>
+          <span class="iconify" data-icon="logos:telegram" data-inline="false" data-width="35px" data-height="35px"></span>
         </a>
       </div>
     </div>
     <div id="right-tab">
       <div id="pac-card">
         <h1>EatGoWhere</h1>
-        <a id="resize-nav-button">Collapse</a>
+        <a id="resize-nav-button">
```

Comment:
Can we change this to a button instead of a link? makes more sense

Reply:
Thanks for pointing it out, no idea why I started with that